packages/client-sdk-web/src/utils/web-client-utils.ts

```diff
@@ -2,11 +2,26 @@ import {CredentialProvider} from '@gomomento/sdk-core';
 
 export function convertToB64String(v: string | Uint8Array): string {
   if (typeof v === 'string') {
```

Comment:
I think it would be a lot easier to reason about this and to make sure we have confidence in it if we added some unit tests.

Also, are there places where this is called where we genuinely don't know whether `v` is a `string` or `Uint8Array`? it seems to me like the logic for those two cases is so wildly different that it'd be better to have two separate functions, and I'd kinda hope that at the call sites for this that we would know which one we were dealing with but maybe not if this is getting called for a lot of cache values when we are about to send the data up to the server.